imports/voting/ui/pages/SingleProposal.vue

```diff
@@ -1,10 +1,9 @@
 <template>
-  <div class="modal">
-    <div class="overlay" @click="toggleProposalModal" />
-    <div class="modal-cont">
-      <div class="modal-header">
-        <button @click="toggleProposalModal">Close</button>
-      </div>
+  <div>
+    <div class="single-proposal" v-if="proposal">
+      <router-link to="/">
+        <i class="fa fa-fw fa-chevron-left" />Back
```

Comment:
This button redirects to root, it doesn't preserve chosen sorting. Maybe better to rename it to "To proposals"?